umbraco-cloud/product-upgrades/minor-upgrades.md

```diff
@@ -1,60 +1,69 @@
 ---
-description: This article explains how Minor upgrades on Umbraco Cloud works.
+description: This article explains how Minor upgrades work in Umbraco Cloud.
 ---
 
 # Minor Upgrades
 
-When a new minor version is released, there are two ways of upgrading your project. For existing projects, you can opt-in for automatic minor upgrades.
+When a new minor version is released, there are two ways to upgrade your project:
 
-To enable Automatic Minor Upgrades follow these steps:
+- [Automatic Minor Upgrades](#automatic-minor-upgrades)
+- [Manual Minor Upgrades](#manual-minor-upgrades)
+
+## Automatic Minor Upgrades
+
+To enable automatic minor upgrades, follow these steps:
 
 1. Go to your Umbraco Cloud project.
-2.  Go to `Settings` -> `Automatic Upgrades`.
+2. Navigate to **Configuration** -> **Automatic Upgrades**.
 
     <figure><img src="../.gitbook/assets/image (47).png" alt="Settings Umbraco Cloud"><figcaption><p>Settings Umbraco Cloud</p></figcaption></figure>
-3.  Enable Automatic Minor Upgrades
+3. Enable **Automatic Minor Upgrades**.
 
     <figure><img src="../.gitbook/assets/image (10) (1).png" alt=""><figcaption><p>Enable Minor Upgrades</p></figcaption></figure>
 
-With automatic upgrades enabled it will be applied to all products on Umbraco Cloud: Umbraco CMS, Umbraco Forms, and Umbraco Deploy. Your project does not need to be running the latest minor for automatic upgrades to work. The project will be upgraded to the latest minor when it is released.
+With automatic upgrades enabled, it will be applied to all products on Umbraco Cloud: Umbraco CMS, Umbraco Forms, and Umbraco Deploy. Your project does not need to be running the latest minor version for automatic upgrades to work. The project will be upgraded to the latest minor version when it is released.
```

Comment:
Was a little confused about the "it will be applied", so attempted a rewrite of this sentence. Let me know what you think!

```suggestion
With automatic upgrades enabled, all products on Umbraco Cloud will automatically be upgraded. This includes Umbraco CMS, Umbraco Forms, and Umbraco Deploy. Your project does not need to be running the latest minor version for automatic upgrades to work. The project will be upgraded to the latest minor version when it is released.
```